app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -279,7 +279,7 @@ def self.raw_alerts_connect(hostname, port, username, password)
         :username => username,
         :password => password
       }
-      ::Hawkular::Alerts::AlertsClient.new(url, credentials)
+      ::Hawkular::Alerts::Client.new(url, credentials)
```

Comment:
@lucasponce I'm not sure I'm seeing it set: https://github.com/ManageIQ/manageiq/blob/b7933dcfad4bbe28eea1bfa41cd3cfecc1ee95d0/app/models/manageiq/providers/hawkular/middleware_manager.rb#L275